src/zookeeper/deploy/start.sh

```diff
@@ -24,14 +24,14 @@ pushd $(dirname "$0") > /dev/null
 kubectl apply --overwrite=true -f zookeeper.yaml || exit $?
 
 
-PYTHONPATH="../.." python -m  k8sPaiLibrary.monitorTool.check_node_label_exist -k zookeeper -v "true"
+PYTHONPATH="../../../deployment" python -m  k8sPaiLibrary.monitorTool.check_node_label_exist -k zookeeper -v "true"
```

Comment:
@YitongFeng To ensure, whether this line could be removed or not.

Reply:
could removed